src/vsg/CMakeLists.txt

```diff
@@ -246,13 +246,19 @@ if(MSVC)
     endmacro()
 
     SET_OUTPUT_DIR_PROPERTY(vsg "")
-	
+
     option(ENABLE_MP_FLAG "Turning on this option will add the multi-processor flag in MSVC for VSG and it's included projects" ON)
-	
+
     if(ENABLE_MP_FLAG)
         target_compile_options(vsg PRIVATE "/MP")
     endif()
 
+    option(DISABLE_CHECKED_ITERATORS "Turning on this option will disable checked iterators in debug mode for visual studio. All imported projects must adhere to this requirement")
```

Comment:
There isn't a default option supplied, what will cmake use in this case?

Reply:
OFF is the always the default

https://cmake.org/cmake/help/v3.0/command/option.html

Reply:
Thanks for the clarification.  Could you add OFF to the option line, I think it's better to explicitly set defaults so that users reading the cmake scripts can read default behaviour without knowing cmake in deoth,  Thanks,

Reply:
Agreed. The PR should be updated with OFF.